lab-michael/test/queue-test.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+let expect = require('chai').expect;
+let Queue = require('../lib/queue.js');
+
+describe('queue model', function() {
+  describe('.enqueue()', function() {
+    it('should throw error if there is nothing in queue', function() {
+      let q = new Queue();
+      let err = new ReferenceError('Nothing in queue');
+      let fn = function(){throw err;};
```

Comment:
This is just a circular test; Create an error, create a function that throws that error, then test that the error and function we created are the same. 

Also, your wording is off a bit, in that we're testing that there's no value passed in as an argument, not if there's nothing in the queue.